fabulaws/library/wsgiautoscale/templates/nginx.conf

```diff
@@ -47,6 +47,11 @@ server {
         proxy_read_timeout 30;
         proxy_connect_timeout 30;
 
+        # rewrite hosts with trailing dot to non-dot version
+        if ($http_host ~ "\.$" ){
+            rewrite ^(.*) $scheme://$host$1 permanent;
```

Comment:
I think it's okay for it to be a redirect, although @kmtracey was the original author of this so she might have a more well-informed opinion. (This was used in production on one project, but lost during the switch to this version of fabulaws.)